Modules/IO/ImageBase/include/itkImageIOBase.h

```diff
@@ -250,9 +250,6 @@ class ITKIOImageBase_EXPORT ImageIOBase : public LightProcessObject
   itkGetEnumMacro(ComponentType, IOComponentEnum);
 
   /** get the type_info for the current pixel component type.
-   * This function is DEPRECATED and only provided for backwards
-   * compatibility.  There is no use for this method that can't
-   * be satisfied by calling GetComponentType.
    */
   virtual const std::type_info &
   GetComponentTypeInfo() const;
```

Comment:
This does not seem to be used by anything within the main repository. Do you have some code which uses it? If not, we could indeed remove this method now, as it has been deprecated for 14 years.

Reply:
@dzenanz OK, I just stopped using `GetComponentTypeInfo()` in elastix: https://github.com/SuperElastix/elastix/pull/1336 https://github.com/SuperElastix/elastix/commit/3119f35b1d12d107eea7a1e8fb7a0909b3de3e84